### ISSUE TITLE
No open source projects are displayed on the home/search page

### ISSUE THREAD

**Issue by fbuys** [codetriage/CodeTriage#1787]
## Expected Behavior

When I visit: https://www.codetriage.com/?language=Go
I expect to see possible Go projects I can contribute to.

When I select any other language,
I expect to see possible Go projects I can contribute to.

## Current Behavior

When I visit: https://www.codetriage.com/?language=Go
I don't see any project
![image](https://github.com/codetriage/CodeTriage/assets/3785596/b5ae2bc8-08a5-4e2c-b65b-df87c68e5717)

Same for any other language I have selected.

## Steps to Reproduce (for bugs)

1. Visit https://www.codetriage.com/?language=Go
2. Notice there are no projects listed
3.
4.

## Context

I am not able to search for a project to contribute to.

## Your Environment

* Version used: One in projeduction
* Browser Name and version: Firefox and Chrome
* Operating System and version (desktop or mobile): Mac
* Link to your project: n/a

**Comment by schneems** [comment]
Thanks for the report. My Redis instance was upgraded automatically (yay!) but I ignored the warning emails (boo) and now I'm hitting a SSL connection issue.

One way that CodeTriage optimizes for performance is by assuming any issue not updated within 24 hours is closed then we only need to make N API requests for M repos where we pull in the open issues. Anything not updated is assumed to be closed. 

The infrastructure that does this runs on Redis (Sidekiq) so basically since it cannot talk to Redis it cannot enqueue new issues and since no new issues are enqueued it assumes all those not updated (all issues) have been closed. The fix is to update redis connection config and re-enqueue all issues. I'll get on that ASAP
